ui/pages/groups/index.js

```diff
@@ -181,6 +183,7 @@ function Details({ group, admin, onDelete }) {
               )}
               <IdentityList
                 list={users}
+                actionText='Remove'
```

Comment:
We don't need the `Remove` here - it is set in the `IdentityList` component as the default value